### DISCUSSION TITLE
Does this work with the Steamdeck?

### DISCUSSION THREAD

**Discussion by martin-schoen** [m4dEngi/RemotePlayWhatever#76] [General]
Can I run it on the Steamdeck to invite another Steamdeck user?

If that works, it enables the possibility to play Emulator games with 2 players, that would be so cool. As long as you have friends that own a Steamdeck too.

**Comment by m4dEngi** [comment]
It does. There's appimage release that can be run on deck since version 0.1.17 (works in desktop mode only) or you can give UI rework preview release a try (can be added as a non-steam game and used in game mode).

**Reply by JLDehesa** [reply]
I cannot find a method to run Remote Play Whatever.

**Comment by AnjuNegaum** [comment]
Hello there, how's everything? 
Before I get on with my issue I just wanna say thanks for the work you've done with this program, I've used it on windows for a while now and I've had a lot of fun with it. However, now that I've acquired a steam deck, I would like to use your program again. Being a lifetime user of windows though, I quite literally can't even begin to understand the set up instructions. I looked for a video guide on youtube, to no avail. I reckon it still will be some time before RemotePlayWhatever is as easy to install as any Decky plugin, so I ask, could you please make a step by step video guide on how to set it up on Linux and get it to work on gaming mode?

**Reply by moraroy** [reply]
@AnjuNegaum @martin-schoen @m4dEngi  i know how to make it work, can i add it to NonSteamLaunchers? https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck
